tutorials/tour/abstract-types.md

```diff
@@ -5,11 +5,10 @@ title: Abstract Types
 disqus: true
 
 tutorial: scala-tour
-num: 2
-outof: 35
```

Comment:
I was just putting the `outof` in the first file alphabetically in the folder since, based on the filename, it wasn't clear which number which article got.